README.md

```diff
@@ -15,3 +15,10 @@ A 2D dungeon crawler with an isometric view. The player starts at the bottom of
 * Camera: 2D Isometric View
 ### Theme
 Travel through different time periods, collecting different weapons to defeat the scaly creatures inside the tower to reach the treasure at the top. 
+
+## Build Requirements
+Unity Version: 2022.3.11f1
+VS Code with Unity Extension or Visual Studio
+
+## Steps to Set Up Project
+If you would like to use VS Code as your editor then first have that installed otherwise, install the Unity Hub first. Once the Unity Hub is installed, install Unity version 2022.3.11f1. When installing, choose Visual Studio or VS Code as your default script editor, this will install Visual Studio automatically if it is not already installed. Then, I would recommend installing Github Desktop. Through Github Desktop, clone this repository into a folder on your computer. This will allow you to then open the project folder on Unity Hub.  
```

Comment:
2022.3.11f1 was not an option in the hub install list. I'm sure any LTS version would be alright, but if 2022.3.11f1 is required, I'd link it to its [installer](https://unity.com/releases/editor/qa/lts-releases).

Otherwise, this was enough for me to figure out how to get my environment set up, so it should suffice for now. Any gaps in knowledge at this point is from unfamiliarity with Unity and doesn't need handholding through this README, in my opinion.

Reply:
I'll approve one comment is resolved.